core/domain/feedback_services_test.py

```diff
@@ -344,6 +344,27 @@ def test_get_thread_summaries(self):
         # Check if the number of unread messages is equal to 1.
         self.assertEqual(number_of_unread_threads, 1)
 
+    def test_get_thread_summaries_increments_message_count(self):
```

Comment:
This name is misleading. Get thread summaries shouldn't "increment" the message count right? I think you meant it gets the correct counts.

Reply:
Done